infrastructure/adminservices-test/altinn-monitor-test-rg/k6_tests_rg_k8s.tf

```diff
@@ -31,7 +31,7 @@ resource "azurerm_monitor_data_collection_rule" "k6tests" {
         "dataCollectionSettings" : {
           "interval" : "1m",
           "namespaceFilteringMode" : "Include",
-          "namespaces" : ["dialogporten", "correspondence"] # Only in the namespaces we have k6 tests running for now.
+          "namespaces" : ["platform", "dialogporten", "correspondence", "core"] # Only in the namespaces we have k6 tests running for now.
```

Comment:
Hijacking the PR and sneaking in the platform namespace as well.
I tend to just `kubectl logs` directly since I usually validate other stuff in cluster but I guess it might be useful when I want to improve the Azure Monitor Integration with Grafana.